boards/xtensa/esp32s2_saola/esp32s2_saola.dts

```diff
@@ -120,8 +120,11 @@
 	pinctrl-names = "default";
 };
 
+
 &flash0 {
 	status = "okay";
+	reg = <0x0 DT_SIZE_M(4)>;
+
```

Comment:
This is already inherited from `esp32s2_wrover_n4r2.dtsi`

Reply:
Thanks for pointing this out! Fixed.

Reply:
Nope, this is still part of `soc: xtensa,riscv: esp32xx: refactor folder structure`.